app/src/main/res/layout/activity_main.xml

```diff
@@ -20,6 +20,5 @@
         android:layout_height="match_parent"
         android:layout_gravity="start"
         android:fitsSystemWindows="true"
-        app:headerLayout="@layout/nav_header_main"
```

Comment:
not criticism, just curiosity- could you please help me understand why this is being removed?